src/index.js

```diff
@@ -21,6 +21,13 @@ import {getPlayers, getPlayer} from './proxy';
 import {cast} from './common/cast';
 // Import playlist
 import {playlist} from './common/playlist';
+// Import plugin framework
+import {BasePlugin} from './common/plugins/base-plugin';
```

Comment:
We didn't do it so far, but I would suggest to add index.js under each logic module (plugins, ads, etc...) and export all of its components from there. This is a good practice and very handy when you need to import multiple components from a certain module. You can then just do `export * from './common/plugins'`

Reply:
done